src/webhook_service/list_webhooks/webhook_repository.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import os
+
+import boto3
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+from boto3.dynamodb.conditions import Key
+
+
+class WebhookRepository:
+    def __init__(self):
+        """
+        Initialize the repository with the DynamoDB table.
+        """
+        self.dynamodb = boto3.resource("dynamodb")
+        self.table = self.dynamodb.Table(os.getenv("DYNAMODB_TABLE"))
+
+    def get_data(self, webhook_type: str, limit: int = 10, sort_order: str = "ASC", page: int = 1, start_key: int = 0, end_key: int = 0):
```

Comment:
The `limit` and `page` are unused in the databased interaction

I suggest simplifying the current implementation by removing the limit and page parameters for now, as pagination is being handled outside the database query

If optimization is required later, you can address this in [SCRUM-200](https://aws-educate-cloud-ambassador-dev-team.atlassian.net/browse/SCRUM-200)